Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -7872,9 +7872,8 @@ class BBQJIT {
         saveValuesAcrossCallAndPassArguments(arguments, callInfo);
 
         if (m_info.isImportedFunctionFromFunctionIndexSpace(functionIndex)) {
-            m_jit.move(TrustedImmPtr(Instance::offsetOfImportFunctionStub(functionIndex)), wasmScratchGPR);
-            m_jit.addPtr(GPRInfo::wasmContextInstancePointer, wasmScratchGPR);
-            m_jit.loadPtr(Address(wasmScratchGPR), wasmScratchGPR);
```

Comment:
Let's also add `static_assert(maxImports < some-reasonable-size);` to keep this invariant.